### DISCUSSION TITLE
PC/Xbox Series X Controller: Menu navigation skips items

### DISCUSSION THREAD

**Discussion by Samhayne** [diasurgical/DevilutionX#7128] [Q&A] [answered]
Hi,

unfortunately when navigating the game menus with my Standard Xbox Series Controller every other menu item will be skipped. Like the buttons were too sensitive.

When using the dpad, the selection will always jump up/down by TWO items.
With the analog stick the same, unless I move it to the right and veeeery slightly down.

I don‘t see any controller sensitivity setting.

Someone got a hint for me?

**Comment by StephenCWills** [comment]
Is your driver registering the controller as two devices simultaneously?

**Reply by Samhayne** [reply]
> Is your driver registering the controller as two devices simultaneously?

Hi Stephen. Hm, good question.
I checked the Windows control panel... among the devices I only find a single "Xinput compatible HID device".
Also when executing joy.cpl via Win+R I only see a single gamepad device.

Please also see the log in the answer below...

**Reply by StephenCWills** [reply]
That sounds pretty definitive. I don't suppose you have any software running that could be emulating keyboard inputs based on controller inputs?

**Reply by Samhayne** [reply, marked as answer]
Ha! That brought me on the right track.
After a bit of googling for what you described I read that Steam could be the culprit.
And indeed… immediately after stopping steam.exe the problems went away.

I already shut down my PC, but I bet it‘s the same „feature“ that is described here:
https://www.reddit.com/r/RetroArch/comments/yuly2u/xbox_controller_inputs_being_read_as_keyboard/

Thank you, Stephen!

**Comment by AJenbo** [comment]
What version? What operating system? Do you have something that maps it as a generic controller?

What controllers does the game detect when you start it up? (it writes this in stdout)

**Reply by Samhayne** [reply]
Hi @AJenbo.
I used the most recent version, 1.5.2, for x86_64 running on my Windows 11 machine.
As mentioned, the controller is an Xbox Series Wireless Controller. 
I doubt I ever installed anything beyond the (automatically found) standard drivers and the Xbox Accessories.

Here's the log output I get at the start:

```
INFO: Adding joystick 0: Xbox Series X Controller
INFO: Opening game controller for joystick at index 0
INFO: Opened game controller with mapping:
0300509d5e040000120b000000007200,*,a:b0,b:b1,x:b2,y:b3,back:b6,guide:b10,start:b7,leftstick:b8,rightstick:b9,leftshoulder:b4,rightshoulder:b5,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a2,righty:a3,lefttrigger:a4,righttrigger:a5,crc:9d50,platform:Windows
```
